cypress/components/password/password.cy.tsx

```diff
@@ -208,27 +172,22 @@ context("Tests for Password component", () => {
       icon()
         .should("be.visible")
         .and("have.css", "color", colorsUtilityMajor300);
-      icon()
-        .getDesignTokensByCssProperty("color")
-        .should(($el) => {
-          expect($el[3]).to.equal("--colorsUtilityMajor300");
-        });
+      icon().getDesignTokensByCssProperty("color").as("colorToken");
+
+      cy.get("@colorToken").its("3").should("equal", "--colorsUtilityMajor300");
     });
 
     it("icon color is 'colorsActionMajorYang300' onHover'", () => {
```

Comment:
**suggestion:** (non-blocking): Think this was actually my fault originally, but may be worth renaming this test during this refactor. As test description references `colorsActionMajorYang300`, but we're actually testing against `colorsUtilityMajor300`